DemocracyClub/UK-Polling-Stations
# Add var to show/hide voter ID message

Now that the English elections are over, we want to hide the voter ID messaging as Northern Ireland has its own vibe.

I've implemented this as an env var so that it can easily be toggled on/off as and when we want to display that messaging. Homepage template has been updated to leverage this, and I've set it to `False` for now (with a default of `False` too).

Docs have been updated to add instructions for this, and I fixed up the docs relating to charismatic election dates whilst I was in there. 

To test locally:
- Run the project
- The homepage shouldn't have any additional cards about ID messaging
- Set `SHOW_GB_ID_MESSAGING` to `True` in your `local.py`
- Reload the homepage
- You should see ID messaging


```[tasklist]
### PR Checklist

- [x] Did I explain all possible solutions and why I chose the one I did?
- [x] Update any documentation
```
